src/preflightCheck.js

```diff
@@ -1,7 +1,16 @@
 import { join } from 'path';
-import { transform } from 'babel-core';
-import { INLINE, RUNTIME, BUNDLED } from './constants.js';
-import importHelperPlugin from './helperPlugin.js';
+import { transform } from '@babel/core';
+import { INLINE, RUNTIME, EXTERNAL } from './constants.js';
+
+function fallbackClassTransform () {
```

Comment:
Can this be killed now? The comment that got removed said the class stuff worked in Babel 7.x

Reply:
comment was about `replaceWith` wrapper/helper function which was handling differences between babel6 and 7

this whole `preflightCheck` is here because it runs a sample code with babel's config and inspect the output to see what kind of helpers did it use - but not every babel config specifies class transform, so if the input code stays untouched it adds this `fallbackClassTransform` to the config and run the check once more

Reply:
Alright I'll accept this as fine for now since it was already doing this anyway, but let it be known that I really think this preflight logic is gross :P